src/components/ValidationMessage/CorrectValidation.tsx

```diff
@@ -0,0 +1,21 @@
+import CorrectIcon from './CorrectIcon'
```

Comment:
This goes for this component and the `CorrectIcon`. This might be picky but I find it better to use the word `Success` instead of `Correct` for both components